src/CarbonAware.WebApi/test/unitTests/CarbonAwareControllerTests.cs

```diff
@@ -111,6 +112,26 @@ public async Task GetForecast_SuccessfulCallReturnsOk(params string[] locations)
         aggregator.Verify(a => a.GetCurrentForecastDataAsync(It.IsAny<Dictionary<string, object>>()), Times.Once);
     }
 
+    /// <summary>
+    /// Tests that successfull call to the aggregator with any data returned results in action with OK status.
+    /// </summary>
+    [TestCase("Sydney", "2022-03-07T01:00:00", "2022-03-07T03:30:00")]
```

Comment:
🐑 you can also include the summary in the test cast name. I.e., "SuccessfulCallReturnsData" or something of the like

Reply:
We could yes. This was directly ported from _SciScoreControllerTests_, and we moved the "1 set of test params" to be as inputs, rather than leave them "inside" the test as the prior instance did. 
Will do.